content/websockets/gateways.md

```diff
@@ -20,46 +22,70 @@ $ npm i --save @nestjs/websockets @nestjs/platform-socket.io
 
 #### Overview
 
-In general, each gateway is listening to the same port as **HTTP server** is running on, unless your app is not a web application, or you have changed the port manually. We can change this behavior by passing an argument to the `@WebSocketGateway(80)` decorator where `80` is a chosen port number. Additionally, you can set a [namespace](https://socket.io/docs/rooms-and-namespaces/) used by this gateway with the following construction:
+In general, each gateway is listening to the same port as **HTTP server** is running on, unless your app is not a web application, or you have changed the port manually. This default behavior can be modified by passing an argument to the `@WebSocketGateway(80)` decorator where `80` is a chosen port number. You can also set a [namespace](https://socket.io/docs/rooms-and-namespaces/) used by the gateway using the following construction:
 
 ```typescript
 @WebSocketGateway(80, { namespace: 'events' })
 ```
 
-> **Warning** The gateway won't start until you put it inside the `providers` array.
+> **Warning** Gateways won't be instantiated until you put them inside the `providers` array of any existing module.
 
 The `namespace` is not a sole available option. You can pass any other property that is mentioned [here](https://socket.io/docs/server-api/). Those properties will be passed to the socket constructor during the instantiation process.
```

Comment:
You can pass any supported [option](https://socket.io/docs/server-api/) to the socket constructor with the second argument to the `@WebSocketGateway()` decorator, as shown below: